### ISSUE TITLE
Message filters

### ISSUE THREAD

**Issue by Geo1088** [r-anime/discord-mod-bot#2]
Needs support from Yuuko for modifying the default command handling flow

**Comment by Geo1088** [comment]
no longer blocked on yuuko, just need to get the interface for configuring these done and then set up the bot process to read them

**Comment by Geo1088** [comment]
actual filtering logic is pretty much there! there's a command that lets you add new `content containsText` filters, though it's not behind any permissions and is meant to be temporary. closing this; will make a new issue for better configuration via the web dashboard.
